package.json

```diff
@@ -1,4 +1,113 @@
-{	
-  "private": true,	
-  "workspaces": ["collect-package-metadata", "service"]	
-}
\ No newline at end of file
+{
+  "name": "package-analyzer",
+  "private": "true",
+  "repository": "git@github.com:Hermanya/package-analyzer.git",
+  "version": "0.1.8",
+  "description": "Package analyzer 🔎 📦",
+  "workspaces": [
+    "packages/*"
+  ],
+  "engines": {
+    "node": ">10.0.0"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*": [
+      "pretty-quick --staged"
+    ],
+    "*.ts": [
+      "eslint --fix"
+    ]
+  },
+  "prettier": {
+    "singleQuote": true,
+    "semi": true,
+    "printWidth": 120
+  },
+  "eslintConfig": {
```

Comment:
you might have noticed that I'm using https://github.com/formium/tsdx which comes with a lot of default configuration. How do these overrides work together?

Reply:
so, In general I remove tsdx given that is too bulky and slow I just this eslint config that is pretty basic, but really strict.

Reply:
I respectfully disagree. 

I'm not sure what you found bulky and slow about it, but I like the "zero-config" nature of this tool. 
Not only it's a huge productivity boost. But it's also very widely adopted and liked within the industry, and endorsed by the typescript team.

It's very much like `create-react-app` (or rather `react-scripts`), in that it abstracts away the configuration of a typical project. Be it react app or a typescript library. The cost of maintenance is dramatically reduced, if you are willing to stay within the "standard configuration", which I intend to.

My plan is to use `tsdx` for the library parts of this project and CRA for the future UI.